blocks/v2-footer/v2-footer.css

```diff
@@ -0,0 +1,413 @@
+    border-top: 1px solid var(--c-tertiary-cool-gray);
+    width: 100%;
+  }
+
+  .v2-footer .v2-footer-links-section {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+  }
+
+  .v2-footer .v2-footer-links-section ul {
+    min-width: 50%;
+  }
+  
+  .v2-footer .v2-footer-form-header .v2-footer__title {
+    font-family: var(--ff-body);
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px;
+    letter-spacing: 1.92px;
+    text-transform: uppercase;
+    margin-bottom: 12px;
+  }
+
+  .v2-footer .v2-footer-main-content {
+    display: grid; 
+    grid-template-columns: 1fr; 
+    grid-template-rows: 1fr 1fr 1fr 1fr; 
+    gap: 0 0; 
+    /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
+    grid-template-areas: 
+      "v2-logo"
+      "v2-content"
+      "v2-form"
+      "v2-social"; 
+      height: 72vh;
+  }
+  
+  .v2-social { 
+    grid-area: v2-social;
+    margin-top: 0;
+  }
+
+  .v2-content { grid-area: v2-content; }
+
+  .v2-form { 
+    grid-area: v2-form;
+    height: 15vh;
+  }
+
+  .v2-logo { 
+    grid-area: v2-logo; 
+    height: 15vh;
+  }
+
+  .v2-footer .v2-footer-links-section ul li {
+    margin-bottom: 14px;
+  }
+
+  /* desktop   */
+  @media (min-width: 1200px) {
+    .v2-footer {
+      padding-top: 96px;
+    }
+  
+    .v2-footer .v2-footer-content {
+      display: flex;
+      flex-flow: row wrap;
+    }
+  
+    .v2-footer .v2-footer-links-section {
+      display: flex;
+      justify-content: space-between;
+      align-items: flex-start;
+      gap: 20px;
+      flex-wrap: nowrap;
+    }
+
+    .v2-footer .v2-footer-links-section ul {
+      min-width: 0;
+      width:160px;
+    }
+
+    .v2-footer .v2-footer-links-section ul li{
+      margin-bottom: 14px;
+    }
+  
+    .v2-footer .v2-footer-links-col {
+      width: 185px;
+    }
+  
+    /* stylelint-disable-next-line no-descending-specificity */
+    .v2-footer .v2-footer-bottom-section li {
+      padding: 10px;
+      width: auto;
+    }
+  
+    .v2-footer .v2-footer-links-col-1 ul li {
+      width: 100%;
+    }
+  
+    .v2-footer .v2-footer-bottom-section {
+      width: 100%;
+      margin-top: 65px;
+    }
+  
+    .v2-footer .v2-footer-bottom-section ul {
+      width: 100%;
+      display: flex;
+      flex-wrap: nowrap;
+      max-width: unset;
+      align-items: flex-start;
+      justify-content: left;
+      margin-top: 22px;
+    }
+
+    /* stylelint-disable-next-line no-duplicate-selectors */
+    .v2-footer .v2-footer-main-content{
+      display: grid; 
+      grid-template-columns: 1fr 1fr 1fr; 
+      grid-template-rows: 1fr 1fr; 
+      gap: 0 0; 
+      /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
+      grid-template-areas: 
+      "v2-logo v2-content v2-form"
+      "v2-social v2-content v2-form";
+      height: 15vh;
+    }
+
+    .v2-social { 
+      margin-top: -80px;
```

Comment:
Unnecessary negative margin. You can get rid of `height: 15vh` for `v2-logo` to solve the problem of positioning elements

Reply:
Updated